thereadingmachine/scraper/news_scraper/settings.py

```diff
@@ -15,7 +15,12 @@
 NEWSPIDER_MODULE = 'thereadingmachine.scraper.news_scraper.spiders'
 
 
+# Logging
 LOG_STDOUT = False
+LOG_LEVEL = 'ERROR'
```

Comment:
a bit harsh as a default, consider using environment variables instead and defaulting to something milder.